cozy/jobs.py

```diff
@@ -66,11 +109,13 @@ def run(self):
     """
 
     def __init__(self):
-        self._thread = Process(target=self._run, daemon=True)
-        self._flags = Array("b", [False] * 3)
+        self._thread = _multiprocessing_context.Process(target=self._run, daemon=True)
+        self._flags = _multiprocessing_context.Array("b", [False] * 4)
```

Comment:
In retrospect, I felt like the constant indexing can be quite confusing -- I propose to refactor this to a dictionary or define some "constant" variables like `HANDLE_SIGINT_GRACEFULLY = 3`